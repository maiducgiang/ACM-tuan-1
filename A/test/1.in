5
A E
C H
E Y
F D
A A