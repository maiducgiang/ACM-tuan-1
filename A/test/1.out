A => B => C => D => E
C => D => H
E => D => H => Y
F => B => C => D
A => A
